pages.it/common/more.md

```diff
@@ -0,0 +1,32 @@
+# more
+
+> Aprire un file per la lettura interattiva, con scorrimento e ricerca.
+> Maggiori Informazioni: <https://manned.org/more>.
+
+- Per aprire un file:
```

Comment:
I'm a Spanish speaker and I think you wrote `to open` instead of `open` in this example description
```suggestion
- Apri un file:
```

Reply:
Is any reason to prefer the indicative form to the infinitive? I have no problem changing it, but it's not necessary, as they're both grammatically correct.

Reply:
:thinking: I'll try to explain this in English lol. We want to describe what the command does when the user run it, so I think indicative is the mood what applies to that.